packages/RichTextEditing-Tests.package/RichTextEditorImportExportTests.class/instance/testUS87ExportAsPlainTextExistsInDialog.st

```diff
@@ -0,0 +1,8 @@
+acceptance-tests
+testUS87ExportAsPlainTextExistsInDialog
+
+	| answer |
+	answer := self answer: RichTextPlainTextExportStrategy
+							inDialog: [self editorModel selectExportStrategy].
+							
+	self assert: answer ~= nil
```

Comment:
Don't we have a notNil check or something like that?

Reply:
no